indico/modules/events/abstracts/__init__.py

```diff
@@ -142,7 +142,7 @@ def _get_notification_placeholders(sender, **kwargs):
 def _extend_editing_menu(sender, event, **kwargs):
     if event.has_feature('abstracts'):
         yield SideMenuItem('abstracts', _('Call for Abstracts'), url_for('abstracts.call_for_abstracts', event),
-                           section='organization')
+                           section='workflows')
```

Comment:
This one is a separate menu. Do we even need the section here? cc @panagiotappl

Reply:
I don't think we need the section here, I will remove it fully.

Reply:
Yeah, we don't at least for now that it's so small.